boards/qorvo/decawave_dwm3001cdk/Kconfig

```diff
@@ -0,0 +1,11 @@
+# DecaWave DWM3001CDK board configuration
+
+# Copyright (c) 2024 The Zephyr Project Contributors
+# # SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DECAWAVE_DWM3001CDK
+config BOARD_SERIAL_BACKEND_CDC_ACM
+	bool "Use USB CDC as serial console backend"
+	default n
```

Comment:
`default n` can go because n is default, but I assume you want this to be `y` so it's usable?

Reply:
It fails tests either way, so I dunno which to do

Reply:
This board will never pass tests using twister because it would need a dedicated real UART, not a USB one that requires the application to be running to create and that vanishes when it stops (or crashes, or encounters an error, etc.), but if you load the test to the device and monitor the USB CDC line output, it should show success, though you might need to add a delay to the test so you can open the port before it outputs it